pkg/kopia/cmd/commands.go

```diff
@@ -0,0 +1,486 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
```

Comment:
Should we rename it to cmdwrapper? Might make more sense.

```suggestion
package cmdwrapper
```

Reply:
I think we should avoid `cmd` package to avoid confusion. I think it's a convention to use `cmd` package for binaries being built. I would name it `command` or `commandwrapper`